### ISSUE TITLE
Fan mode & Dry mode

### ISSUE THREAD

**Issue by luke-b5114** [yenoiwesa/homebridge-daikin-airbase#55]
Evening, 

Just installed this plug in and working awesome !! 
But is there any way to delete the "Switches" in Homekit for Fan mode and Dry mode. As these are not required, i do not need them displayed.

Thank so much

**Comment by mdiberardino-atlassian** [comment]
Yes there is! Have a look at the README file, you can set in your settings the following options:
```json
 {
  "platform": "DaikinAirbase",
  "name": "Daiking Airbase",
  "overrides": {
      "fanModeSupported": false,
      "dryModeSupported": false
   }
}
```

Set these in your JSON settings file and restart homebridge, and these switches should be gone!
Let me know.

**Comment by luke-b5114** [comment]
Switches are still there....
unsure if thats an error on my behalf.

are you able to look over this please ?

{
            "name": "Daikin Airbase",
            "pollingInterval": 5,
            "useIndividualZoneControls": false,
            "platform": "DaikinAirbase",
            "overrides": {
                "fanModeSupported": false,
                "dryModeSupported": false
            }

**Comment by mdiberardino-atlassian** [comment]
You're missing a closing bracket at the end but I assume it's just a bad copy paste and not the actual configuration being broken. You might want to try to clear the accessory cache too.

**Comment by luke-b5114** [comment]
Awesome !! 
Just had to clear the accessory cache.

Appreciate the help and quick replies.
